bigbluebutton-html5/.meteor/packages

```diff
@@ -3,24 +3,28 @@
 # 'meteor add' and 'meteor remove' will edit this file for you,
 # but you can also edit it by hand.
 
-standard-app-packages@1.0.9
-arunoda:npm@0.2.6
-amplify
-blaze@2.1.8
-francocatena:status
-mizzao:timesync
-clinical:nightwatch
-cfs:power-queue
-cfs:reactive-list
-cfs:micro-queue
+meteor-base@1.4.0
+mobile-experience@1.0.5
+mongo@1.6.0
 reactive-var@1.0.11
-ecmascript@0.9.0
+
+standard-minifier-css@1.5.2
+standard-minifier-js@2.4.0
+es5-shim@4.8.0
+ecmascript@0.12.3
+shell-server@0.4.0
+
+static-html
 react-meteor-data
-standard-minifier-css@1.3.5
-standard-minifier-js@2.2.0
-nathantreid:css-modules
-shell-server@0.3.0
-http@1.3.0
-dynamic-import@0.2.0
+http@1.4.1
+session@1.2.0
+tracker@1.2.0
+check@1.3.1
+jquery
+
+nathantreid:css-modules@=3.1.4
 rocketchat:streamer
-session
+cfs:power-queue
+cfs:micro-queue
+cfs:reactive-list
+underscore
```

Comment:
We use Lodash not undescore.
https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/package.json#L45